src/apprt/gtk-ng/global.zig

```diff
@@ -0,0 +1,15 @@
+//! global static definitions
+
+const builtin = @import("builtin");
+
+/// GTK application ID
+pub const application_id = switch (builtin.mode) {
```

Comment:
Let's just put these constants either directly `gtk-ng.zig` or `gtk-ng/App.zig`. I think we should _really_ use `build_config.zig` for this instead because this can be overridden via build configs no? But I'm willing to TODO that.